about.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="./about/aboutMobile.css"
+      media="screen and (max-width: 768px)"
+    />
+    <link
+      rel="stylesheet"
+      href="./about/aboutDesc.css"
+      media="screen and (min-width: 768px)"
+    />
+    <title>About/page/hyundai</title>
+  </head>
+  <body>
+    <header>
+      <nav class="h_mobile mobile-menu">
+        <i class="fas fa-bars menu-bars"></i>
+        <div class="dropMenu">
+          <i class="fas fa-times close-icon"></i>
+          <ul class="menu-list">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#courses">Contact</a></li>
+            <li class="login">
+              <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="top_icons">
+        <ul>
+          <li>
+            <a><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a><i class="fab fa-twitter"></i> </a>
+          </li>
+          <li>Languages</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
```

Comment:
- [ ] Kindly use HTML semantic `nav` instead of `ul`. Using semantic tags is considered one of the best practices in the programming world.